internal/controller/lcore_deployment.go

```
@@ -0,0 +1,493 @@
+	// Llama-stack needs POSTGRES_PASSWORD for ${env.POSTGRES_PASSWORD} substitution in its config
+	llamaStackEnvVars := append(llamaEnvVars,
+		buildPostgresPasswordEnvVar(),
+		corev1.EnvVar{
+			Name:  "LLAMA_STACK_LOGGING",
+			Value: "all=info",
+		},
+	)
+	llamaStackEnvVars = append(llamaStackEnvVars, buildAdditionalCAEnvVars(instance)...)
+
+	llamaStackContainer := corev1.Container{
+		Name:            "llama-stack",
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofmt)